static/js/components/forms/ProfileFormFields.js

```diff
@@ -22,6 +22,11 @@ const US_POSTAL_CODE_REGEX = /[0-9]{5}(-[0-9]{4}){0,1}/
 const CA_POSTAL_CODE_REGEX = /[A-Z][0-9][A-Z] [0-9][A-Z][0-9]/
 const COUNTRIES_REQUIRING_POSTAL_CODE = [US_ALPHA_2, CA_ALPHA_2]
 const COUNTRIES_REQUIRING_STATE = [US_ALPHA_2, CA_ALPHA_2]
+export const NAME_REGEX = /^(?![~!@&)(+:'.?/,`-]+)([^/^$#*=[\]`%_;<>{}"|]+)$/
+
+// Field Error messages
+export const NAME_REGEX_FAIL_MESSAGE =
+  "Name cannot start with a special character, and it cannot contain any character from {/^$#*=[]`%_;<>{}}"
```

Comment:
nit:
```suggestion
  "Name cannot start with a special character (LIST OF SPECIAL CHARACTERS), and cannot contain any of (/^$#*=[]`%_;<>{}|)"
```

Reply:
@briangrossman There was just a minor change in the Field validation error message suggested by @HamzaIbnFarooq, I've updated the screenshot in PR. Can I assume that this change in message is good with you too?

Reply:
Sure. That's fine. (and I just tested it on RC)